src/js/internal/sql/postgres.ts

```
@@ -1,12 +1,14 @@
 import type { PostgresErrorOptions } from "internal/sql/errors";
 import type { Query } from "./query";
-import type { DatabaseAdapter, SQLHelper, SQLResultArray, SSLMode } from "./shared";
-const { SQLHelper, SSLMode, SQLResultArray } = require("internal/sql/shared");
+import type { ArrayType, DatabaseAdapter, SQLArrayParameter, SQLHelper, SQLResultArray, SSLMode } from "./shared";
+const { SQLHelper, SSLMode, SQLResultArray, SQLArrayParameter } = require("internal/sql/shared");
```

Biome (2.1.2)

[error] Shouldn't redeclare 'SQLHelper'. Consider to delete it or rename it.

'SQLHelper' is defined here:


(lint/suspicious/noRedeclare)